app/models/cda/defendant.rb

```diff
@@ -3,7 +3,9 @@
 module Cda
   class Defendant < BaseModel
     def self.find_from_id_and_urn(defendant_id, urn)
-      find(:one, from: "/api/internal/v2/prosecution_cases/#{urn}/defendants/#{defendant_id}")
+      find(:one,
+           from: "/api/internal/v2/prosecution_cases/#{make_safe_for_path(urn)}/" \
+                 "defendants/#{make_safe_for_path(defendant_id)}")
```

Comment:
I was thinking, can we sanitize the entire path, instead of urn and defendant_id separetely?

Reply:
Sadly not, because different rules apply to identifiers than the rest of the path. E.g. we absolutely have to make sure there are no `/` characters in the URN, as that would cause us to call a different endpoint to the one intended. But there _can_ be slashes in the rest of the path.